tools/update-readme.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @fileoverview Script to update the README with team and sponsors.
+ * Note that this requires eslint/website to be available in the same
+ * directory as the eslint repo.
+ *
+ *   node tools/update-readme.js
+ *
+ * @author Harish Kumar S S
+ */
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+import { readFileSync, writeFileSync } from "node:fs";
+import { stripIndents } from "common-tags";
+import ejs from "ejs";
+import got from "got";
+
+//-----------------------------------------------------------------------------
+// Data
+//-----------------------------------------------------------------------------
+
+const SPONSORS_URL =
+	"https://raw.githubusercontent.com/eslint/eslint.org/main/src/_data/sponsors.json";
+const TEAM_URL =
```

Comment:
I think we can remove all of the team-related functionality from this file. Let's just focus on sponsors.